src/app/Caches/CacheMetrics.tsx

```diff
@@ -136,19 +137,19 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
       <Card>
         <CardTitle>{t('caches.cache-metrics.data-access-title')}</CardTitle>
         <CardBody>
-          <div style={{height: '300px', width: '70%'}}>
+          <div style={{ height: '300px', width: '70%' }}>
```

Comment:
extract to scss file as a class

Reply:
Do we have a scss file already where we can export?